frame/support/src/traits.rs

```diff
@@ -1128,6 +1128,22 @@ pub trait Currency<AccountId> {
 	) -> SignedImbalance<Self::Balance, Self::PositiveImbalance>;
 }
 
+/// Trait for providing an ERC-20 style set of named fungible assets.
+pub trait Fungibles<AccountId> {
+	/// Means of identifying one asset class from another.
+	type AssetId: FullCodec + Copy + Default;
+	/// Scalar type for storing balance of an account.
+	type Balance: AtLeast32BitUnsigned + FullCodec + Copy + Default;
+	/// Get the `asset` balance of `who`.
+	fn balance(asset: Self::AssetId, who: &AccountId) -> Self::Balance;
```

Comment:
also good to have a `min_balance() -> Self::Balance` function

Reply:
Will introduce in next PR.